docs/cli-reference/ark_snapshot-location_get.md

```diff
@@ -0,0 +1,41 @@
+## ark snapshot-location get
+
+Get snapshot locations
+
+### Synopsis
+
+Get snapshot locations
+
+```
+ark snapshot-location get [flags]
```

Comment:
Should be plural.

Reply:
These files are autogenerated, based on how the command is registered in cobra.

Reply:
Thanks!